src/bindings/js/node/src/tensor.cpp

```diff
@@ -138,3 +139,12 @@ Napi::Value TensorWrap::get_shape(const Napi::CallbackInfo& info) {
 Napi::Value TensorWrap::get_element_type(const Napi::CallbackInfo& info) {
     return cpp_to_js<ov::element::Type_t, Napi::String>(info, _tensor.get_element_type());
 }
+
+Napi::Value TensorWrap::get_size(const Napi::CallbackInfo& info) {
+    if (info.Length() > 0) {
+        reportError(info.Env(), "getSize() does not accept any arguments.");
+        return info.Env().Null();
+    }
+    const auto size = static_cast<double>(_tensor.get_size());
```

Comment:
Why does it cast to double, not uint?

Reply:
Hi @vishniakov-nikolai thanks for your feedback , my reasoning behind casting into `double`  Type was because of it's close alignment with `Number` Type in JS.